app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal():
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
```

Comment:
The `print_name` method is defined to return a `str`, but it actually returns `None` because it only prints a message. Consider changing the return type to `None` or returning a string.